content/en/blog/_posts/2021-08-11-support-for-HostProcess-Containers/index.md.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+title: 'Windows HostProcess Containers'
+date: 2021-08-16
+slug: Windows HostProcess Containers
```

Comment:
Hi, slug needs to change to not have spaces. please see an example here https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#technical-considerations-for-submitting-a-blog-post